studygroups/tests/test_tasks.py

```diff
@@ -80,7 +73,7 @@ def test_dont_generate_reminder_4days_before(self):
         sg.save()
         sg = StudyGroup.objects.get(pk=1)
         generate_all_meetings(sg)
-        generate_reminder(sg)
+        tasks.generate_reminder(sg)
```

Comment:
This could be what causes problems. Generally, prefer specific imports over general imports like https://github.com/p2pu/learning-circles/pull/289/files?utf8=%E2%9C%93&diff=unified&w=1#diff-ea0cbd356b596db10e24ba5944a6fb43R28

Reply:
oh yeah i made that change when i was getting an error about not being able to import "send_message" but then i realized it was because of a circular dependency and never changed it back.